core/server/master/src/main/java/alluxio/master/metastore/heap/HeapInodeStore.java

```diff
@@ -42,7 +42,8 @@
   /**
    * @param args inode store arguments
    */
-  public HeapInodeStore(InodeStoreArgs args) {}
+  public HeapInodeStore(InodeStoreArgs args) {
+  }
```

Comment:
Some places we use `{}` and others we have a line break, any convention?

Reply:
I prefer `{}`